lib/wca_live/scoretaking.ex

```diff
@@ -641,4 +642,21 @@ defmodule WcaLive.Scoretaking do
       %{round: final_round, results: podium_results}
     end)
   end
+
+  @doc """
+  Returns the number of results entered by the given staff member at
+  the corresponding competition.
+  """
+  @spec entered_results_count(%StaffMember{}) :: integer()
+  def entered_results_count(%StaffMember{} = staff_member) do
+    from(result in Result,
+      join: round in assoc(result, :round),
+      join: competition_event in assoc(round, :competition_event),
+      join: competition in assoc(competition_event, :competition),
+      where:
+        competition.id == ^staff_member.competition_id and
```

Comment:
I added competition to the query, because otherwise we would count all results for the given user across all competitions :)